frontend/src/views/consumer/filter-view/index.js

```diff
@@ -0,0 +1,193 @@
+                  highlightColor={theme.palette.primaryDeep}
+                  tags={[
+                    { value: '0-3', label: '0-3' },
+                    { value: '3-6', label: '3-6' },
+                    { value: '6-12', label: '6-12' },
+                    { value: '13+', label: '13+' },
+                  ]}
+                />
+              </div>
+              <div>
+                <Typography type="subheader">Alue</Typography>
+                <TagPillGroup
+                  value={store.filters.area}
+                  onChange={this.setAreaFilter}
+                  highlightColor={theme.palette.primaryDeep}
+                  tags={area.map(a => ({ value: a, label: a }))}
+                />
+              </div>
+              <div>
+                <Typography type="subheader">Päivämäärä</Typography>
+                <TagPillGroup
+                  highlightColor={theme.palette.primaryDeep}
+                  value={store.filters.date}
+                  onChange={this.setDateFilter}
+                  tags={[
+                    {
+                      value: 1,
+                      label: 'Tammikuu',
+                    },
+                    {
+                      value: 2,
+                      label: 'Helmikuu',
+                    },
+                    {
+                      value: 3,
+                      label: 'Maaliskuu',
+                    },
+                    {
+                      value: 4,
+                      label: 'Huhtikuu',
+                    },
+                    {
+                      value: 5,
+                      label: 'Toukokuu',
+                    },
+                    {
+                      value: 6,
+                      label: 'Kesäkuu',
+                    },
+                    {
+                      value: 7,
+                      label: 'Heinäkuu',
+                    },
+                    {
+                      value: 8,
+                      label: 'Elokuu',
+                    },
+                    {
+                      value: 9,
+                      label: 'Syyskuu',
+                    },
+                    {
+                      value: 10,
+                      label: 'Lokakuu',
+                    },
+                    {
+                      value: 11,
+                      label: 'Marraskuu',
+                    },
+                    {
+                      value: 12,
+                      label: 'Joulukuu',
+                    },
+                  ]}
+                />
+              </div>
+              <div>
+                <Typography type="subheader">Tapahtuminen Typpi</Typography>
+                <TagPillGroup
+                  value={store.filters.eventType}
+                  onChange={this.setEventTypeFilter}
+                  highlightColor={theme.palette.primaryDeep}
+                  tags={['Kurssit Ja Työpajat', 'Näyttelyt', 'Esitykset'].map(
```

Comment:
I would also define these tags as a constant somewhere else than inside the render method here..

Reply:
I will also pull this out of the component